src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageProjectModel.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat.ListPackage
+{
+    /// <summary>
+    /// Calculated project data model for list report
+    /// </summary>
+    internal class ListPackageProjectModel
+    {
+        private readonly ListPackageArgs _listPackageArgs;
+
+        internal List<ReportProblem> ProjectProblems { get; } = new List<ReportProblem>();
+        internal string ProjectPath { get; private set; }
+        // Original packages
+        internal IEnumerable<FrameworkPackages> Packages { get; private set; }
+        // Calculated project model data for each targetframeworks
+        internal List<ListPackageReportFrameworkPackage> TargetFrameworkPackages { get; set; }
+        internal string HttpSourceWarning { get; private set; }
```

Comment:
Is this used? 
CTRL + F doesn't show it.

Reply:
You're right, removed, it's happened to last refactoring.